packages/content-conformance/src/__tests__/config.test.ts

```diff
@@ -11,7 +11,16 @@ describe('loadConfig', () => {
     })
 
     expect(config.root).toEqual(fixturePath)
-    expect(config.contentFileGlobPattern).toEqual('content/**/*.mdx')
```

Comment:
@thiskevinwang I don't think we can have a snapshot here due to `root` being resolved to an absolute path.

Reply:
Ah, yea good catch, I missed that. Will revert